step.yml

```diff
@@ -0,0 +1,38 @@
+---
+title: Generate Xcode html report
+summary: The Step converts xcresult summaries to html reports.
+description: |-
+  Fill out.
+website: https://github.com/bitrise-steplib/bitrise-step-generate-xcode-html-report
+source_code_url: https://github.com/bitrise-steplib/bitrise-step-generate-xcode-html-report
+support_url: https://github.com/bitrise-steplib/bitrise-step-generate-xcode-html-report/issues
+
+type_tags:
+- test
+
+project_type_tags:
+- ios
+- react-native
+- cordova
+- ionic
+- flutter
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/bitrise-step-generate-xcode-html-report
+
+inputs:
+- verbose: "false"
```

Comment:
Shouldn't we wire in an input for the test results root dir? This would make e2e testing easier.

Reply:
I wanted to hide it intentionally because the user should not care about it. But then I can see that for example in the deploy step we also expose a bunch of inputs which the user should never modify. So I added an input field for this.